pom.xml

```diff
@@ -19,97 +19,128 @@
 		<fabric8.mode>kubernetes</fabric8.mode>
 	</properties>
 
-	<build>
-		<plugins>
-			<plugin>
-				<groupId>org.apache.maven.plugins</groupId>
-				<artifactId>maven-compiler-plugin</artifactId>
-				<version>3.7.0</version>
-				<configuration>
-					<source>1.8</source>
-					<target>1.8</target>
-				</configuration>
-			</plugin>
+	<profiles>
+		<profile>
+			<id>simple</id>
```

Comment:
You don't need this profile. As the conf is same for both simple and full profiles, you can put it outside profiles section.